Assets/Tests/InputSystem/Plugins/UITests.cs

```diff
@@ -3761,6 +3761,264 @@ public IEnumerator UI_WhenCursorIsLockedToScreenCenter_PointerEnterAndExitEvents
         Assert.That(callbackReceiver.events.Any(e => e.type == EventType.PointerExit), Is.True);
     }
 
+    #region Multi Display Tests
+#if UNITY_INPUT_SYSTEM_ENABLE_MULTIDISPLAY
```

Comment:
We might also want to limit this by the platforms that are currently supported.

WinEditor, MacEditor, LinuxEditor and AndroidPlayer have this implemented.

MacPlayer, WinPlayer, LinuxPlayer, and iOSPlayer do not, and I am concerned that this might raise false red flags when running for those platforms.

Reply:
No problem, I'll add those in.